src/coreclr/vm/dacenumerablehash.inl

```diff
@@ -345,45 +345,47 @@ DPTR(VALUE) DacEnumerableHashTable<DAC_ENUM_HASH_ARGS>::BaseFindFirstEntryByHash
     do
     {
         DWORD cBuckets = GetLength(curBuckets);
+        if (cBuckets > 0)
```

Comment:
I assume we should never get zero here under normal circumstances.

Should this be under DACCESS_COMPILE ifdef to make it clear that this check is DAC hardening for invalid process state?